Makefile

```diff
@@ -0,0 +1,16 @@
+all: test
+
+install-ci:
+	pip install -r requirements/ci.txt
+
+install-local:
+	pip install -r requirements/local.txt
```

Comment:
But bear in mind that `pip install` does not have to know if it's in a virtualenv or not. For the sake of Continuous Integration, this makefile is enough. See the travis outputs on ✅